src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -23,7 +23,7 @@ public static class Keywords
             public const EventKeywords ProcessorCount = (EventKeywords)0x2;
         }
 
-        internal static RuntimeEventSource? Log => new RuntimeEventSource();
+        internal static RuntimeEventSource? Log = new RuntimeEventSource();
```

Comment:
```suggestion
        internal static RuntimeEventSource Log { get; } = new RuntimeEventSource();
```
`Log` is a property name according to the repo naming conventions, so it should be a defined as a property. Also, is there a reason for it to be nullable?

Alternatively, change the name to `s_log`.